README.md

```diff
@@ -47,15 +47,16 @@ In **PowerShell** or in a **Command Prompt** window:
     ```shell
     apt-get install \
         zlib1g-dev libssl-dev libncurses-dev libffi-dev \
-        libsqlite3-dev libreadline-dev libbz2-dev
+        libsqlite3-dev libreadline-dev libbz2-dev build-essential
```

Comment:
Put `build-essential` first.

Reply:
Will do. Do you mean before zlib1b-dev or before libsqlite3-dev?

Reply:
Before `zlib1g-dev`. Everything from `apt-get install` upto `libbz2-dev` is a list spanning 3 lines.